<?xml version="1.0"?>
<launch>

    <node pkg="robot_lib_test" type="obstacle_detection_avoidance_test" name="obstacle_detection_avoidance_test" output="screen" required="true">
        <!-- The parameters below are required for initializing the robot library -->
        <rosparam file="$(find robot_lib_test)/params/robot_lib_params.yaml" command="load"/>

        <!-- The parameters below are only required for test nodes -->
        <param name="enable_plan_path"          type="bool"       value="true" />
        <param name="start_from_current_pose"   type="bool"       value="true" />
        <param name="start_x"                   type="double"     value="0.0" />
        <param name="start_y"                   type="double"     value="0.0" />
        <param name="goal_x"                    type="double"     value="2.0" />
        <param name="goal_y"                    type="double"     value="0.0" />

        <param name="enable_clear_costmap"      type="bool"       value="true" />
    </node>
</launch>
